modules/material/material.service.js

```diff
@@ -5,6 +5,7 @@ const {
   MATERIAL_NOT_FOUND,
 } = require('../../error-messages/material.messages');
 const Currency = require('../currency/currency.model');
+const { calculatePrice } = require('../currency/currency.utils');
```

Comment:
Please remove this line, because it will cause errors (we have it on line 2).
![image](https://user-images.githubusercontent.com/59802190/105636427-0863b680-5e71-11eb-8033-204875e34339.png)